### PR TITLE
Extended Mapping Set-Up

### PR DESCRIPTION
**Pull request by TheGoodBoi411**
This is a revised version of my first (real) PR, so this is just me adding the "Set-Up" section in the "Extended Mapping" guides.

### CODE REVIEW

**Comment by TheGoodBoi411** [comment]
I actually thought of this, and I asked about it. Aeroluna came in said that MMA2 only checks if there is a MappingExtensions.dll file in there, not caring about what version it is. In the guide, that option is only for people who do not have Beat Saber, where the person "tricks" MMA2 into thinking you have mapping extensions. Maybe you do have to update it, but I do not believe you have to.

File: wiki/mapping/extended-mapping.md [hunk 1]
```diff
@@ -10,9 +10,69 @@ _A quick summary of Extended Mapping_
...
+1. Start Beat Saber and then close out after the game finishes loading. This is to ensure the game is ready for mods to be installed.
+2. Open [ModAssistant](https://github.com/Assistant/ModAssistant/releases/latest)
+3. Make sure the version selected in the dropdown is compatible with the Beat Saber version installed on your computer.
+4. Check the checkbox for MappingExtensions. (You can select additional mods to install as well.)
+5. Click Install and Update and make sure there are no errors.
+6. Open Beat Saber again to initialize the mods you have installed then quit the game.
+
+To check if it worked navigate to your Beat Saber install and make sure that a MappingExtensions.dll file is present.
+
+Oculus: `\Oculus Apps\Software\hyperbolic-magnetism-beat-saber\Plugins`
```
Comment (megalon) [inline wiki/mapping/extended-mapping.md]: There is a page in the wiki FAQ that explains where the install folder is located.
To remove redundancy and save space on here, consider replacing this with something like
`Not sure where Beat Saber is installed? See [install folder](`/faq/install-folder.md`)`

**Comment by megalon** [comment]
I made a suggestion above, but aside from that, do you think that this PR is ready to merge?

**Comment by helencarnate** [comment]
Agree with your recommendation and it's ok to push other than that. We're workshopping all remaining content in Google and only doing a PR when it's 100% ready (like Basic Mapping) so this page won't be heavily linked until it's finished.

**Comment by TheGoodBoi411** [comment]
Fixed/Added the needed changes.

**Comment by TheGoodBoi411** [comment]
I think I fixed it.

**Pull request merged**